### ISSUE TITLE
Create a configurable key-sanitizer processor

### ISSUE THREAD

**Issue by rz** [getsentry/raven-python#984]
I need a processor to sanitize financial data (SSN, EIN, tax ids, bank account numbers, ...). The mechanics of the processor should be very similar to those of [SanitizePasswordsProcessor], except operating on different `FIELDS`.

One potential solution to the general version of this problem is to have a processor that sanitizes a set of configurable keys. The keys to sanitize would be specified with an optional setting.

[SanitizePasswordsProcessor]: https://github.com/getsentry/raven-python/blob/c69abaa1506a426802e979c7f8c6a4ab0ef9c4d7/raven/processors.py#L69

**Comment by mattrobenolt** [comment]
So we could refactor `SanitizePasswordsProcessor` slightly into a generic `SanitizeFieldsProcessor` that takes a set of fields as input. Then `SanitizePasswordsProcesses` ultimately subclasses `SanitizeFieldsProcessor` with a hardcoded list of fields.

Does that make sense?

**Comment by rz** [comment]
Yep, that makes sense. I'll want to have tests for `SanitizeFieldsProcessor` and a code review before refactoring the existing `SanitizePasswordsProcessor`.
